### ISSUE TITLE
qaseio - `get_results` failed

### ISSUE THREAD

**Issue by MaxBai-crypto** [qase-tms/qase-python#88]
qaseio 3.1.1

```
filters = GetResultsFiltersParameter(
                    status="failed",
                    run=str(self.test_run_id),
)
result_list: ResultListResponse = results_api_instance.get_results(
                        self.project_code,
                        filters=filters,
                        limit=limit,
                        offset=offset
                    )
```

HTTP response body: {"status":false,"errorMessage":"Data is invalid.","errorFields":[{"field":"filters","error":"The filters must be an array."}]}

**Comment by philipp-kochnov-stenn** [comment]
I have this problem too. :(

**Comment by n3r** [comment]
Hello @philipp-kochnov-stenn, @MaxBai-crypto!

This issue should have been fixed in #101 (https://github.com/qase-tms/qase-python/releases/tag/qaseio-v3.2.0). Can you check, please?
